apps/full-stack-tests/src/RenderUtils.tsx

```diff
@@ -0,0 +1,68 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+
+import { createElement, Fragment, PropsWithChildren, ReactElement, StrictMode } from "react";
+import { ThemeProvider } from "@itwin/itwinui-react";
+import { RenderHookOptions, RenderHookResult, renderHook as renderHookRTL, RenderOptions, RenderResult, render as renderRTL } from "@testing-library/react";
+import { userEvent, UserEvent } from "@testing-library/user-event";
+
+function createWrapper(Outer: React.JSXElementConstructor<{ children: React.ReactNode }>) {
+  return (Inner?: React.JSXElementConstructor<{ children: React.ReactNode }>) => {
+    return Inner ? ({ children }: PropsWithChildren<unknown>) => createElement(Outer, undefined, createElement(Inner, undefined, children)) : Outer;
+  };
+}
+
+function combineWrappers(wraps: Array<ReturnType<typeof createWrapper>>, wrapper?: React.JSXElementConstructor<{ children: React.ReactNode }>) {
+  let currWrapper = wrapper;
+  for (const wrap of wraps) {
+    currWrapper = wrap(currWrapper);
+  }
+  return currWrapper;
+}
+
+function createDefaultWrappers(disableStrictMode?: boolean, addThemeProvider?: boolean) {
+  const wrappers: Array<ReturnType<typeof createWrapper>> = [];
+  if (addThemeProvider) {
+    wrappers.push(createWrapper(ThemeProvider));
+  }
+
+  // if `DISABLE_STRICT_MODE` is set do not wrap components into `StrictMode` component
+  const StrictModeWrapper = process.env.DISABLE_STRICT_MODE || disableStrictMode ? Fragment : StrictMode;
```

Comment:
We don't need add StrictMode wrapper anymore. RTL allows enabling strict mode. e.g. https://github.com/iTwin/presentation/blob/7f354909f0a538c20622d320cc2f34f4eb2c0359/packages/hierarchies-react/src/test/TestUtils.ts#L22

Reply:
Cool, removed the wrapper from components and full stack tests.